content/src/display-content/business-formation/nexus/dba-resolution-foreign.md

```diff
@@ -0,0 +1,18 @@
+---
+urlSlug: DBA-resolution
+filename: dba-resolution-foreign
+name: Get a Resolution Adopting Your DBA Name
+id: DBA-resolution-foreign
+---
```

Comment:
first thought - why does this have all this grey matter here? I tested the app and the url doesn't change across each step of the DBA flow. Why do we defined a url slug and other stuff here?

second thought, more for longer-term thought, but I thought we were trying to move away from "display-content" markdowns and storing this stuff in just a string in the config JSONs anyways. It makes editing in the CMS easier and also makes our lives way easier in coding because we don't have to static import the displayContent, instead we can just import the config directly. I know we still have a few left, but it's worth thinking about not increasing the stack of what is essentially debt that needs to be turned into a config string,